app/views/ops/_rbac_role_details.html.haml

```diff
@@ -19,28 +19,64 @@
             = javascript_tag(javascript_focus('name'))
         .form-group
           %label.col-md-4.control-label
-            = _('Access Restriction for Orchestration Stacks, Key Pairs, Services, VMs, and Templates')
+            = _('Access Restriction for Catalog Items')
```

Comment:
While it is functional, I don't really like the new role view. The phrase "Access Restriction for ..." is repeated five times. This also causes each access restriction label to wrap onto two lines. Yuck 🥴 !

I'm looking into grouping the restriction settings to improve. Suggestions are welcome 😄